### PR TITLE
Restrict REPL usage to compatible FW builds

### PR DESCRIPTION
**Pull request by Hexxeh**

### CODE REVIEW

File: packages/fdb-debugger/src/index.ts [hunk 1]
```diff
@@ -43,6 +43,8 @@ export interface ConsoleTrace {
   message: any[];
 }
 
+const FBOS3_EVAL_QUIRK = /^[a-zA-Z]+ \d+\.31\.1\.\d+$/;
```
Comment (corhere) [inline packages/fdb-debugger/src/index.ts]: Character classes FTW!
```suggestion
const FBOS3_EVAL_QUIRK = /^\w+ \d+\.31\.1\.\d+$/;
```

Comment (corhere) [inline packages/fdb-debugger/src/index.ts]: Please add a doc comment describing what the quirk is, why it is necessary and what workaround is required. Maybe even include the relevant issue number from our internal bug tracker. We will have to maintain this workaround for much longer than we will remember what it's for, so it's important to have it all documented here.

**Review by corhere — CHANGES_REQUESTED** [review]

File: packages/fdb-debugger/src/index.ts (hunk 1 above)
Comment (Hexxeh) [reply packages/fdb-debugger/src/index.ts]: I was intentionally being more strict, but maybe it's unnecessary...

Comment (corhere) [reply packages/fdb-debugger/src/index.ts]: Oops, I thought that `\w` was equivalent. Maybe I'm mixing it up with character classes from another language's regexes.

**Review by corhere — APPROVED** [review]

**Pull request merged**